dev/io.openliberty.checkpoint_fat/publish/servers/checkpointPasswordUtilities/server.xml

```diff
@@ -21,9 +21,10 @@
     </featureManager>
 
     <application type="war" id="DefaultPrincipalMappingApp" name="DefaultPrincipalMappingApp" location="${server.config.dir}/apps/DefaultPrincipalMappingApp.war"/>
-
-    <variable name="auth.user.password" defaultValue="{aes}APb9ZaYzUL+JsfFD/OOBGaPM0evjmx5AnvmzbaKgffyX" /> <!-- APPPWD -->
-    <variable name="default.user.password" defaultValue="{aes}APb9ZaYzUL+JsfFD/OOBGaPM0evjmx5AnvmzbaKgffyX" /> <!-- APPPWD -->
+    
+    <variable name="auth.user.password" defaultValue="{aes}ARA6E+YeyO1II3V5/j78ZPSryIQKz5Hu4OMs6qhFw+rn6IjvU8aHX+fy4hX1ZtAU4HPbRZ5DaHmGb7CpjlXfX1Bm9leNqxgd+q8nvzTNBGs6wW8RHj/n0GaiuT4n7m2pYy0pDDh5amoZ" /> <!-- APPPWD -->
```

Comment:
Could you also update the values configured into the environment for "testUser" and "testPassword" to use `{aes}` here?

https://github.com/OpenLiberty/open-liberty/blob/7eb1c7bd95bd5903653bd617458d393c8ce66f83/dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/PasswordUtilsTest.java#L68-L69

Reply:
yes, will do